src/main/java/org/gridsuite/modification/server/dto/byfilter/simple/BooleanModificationByFilterInfos.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.dto.byfilter.simple;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.server.dto.byfilter.DataType;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
```

Comment:
It may be work also, but centralizing the declaration in the abstraction class is not a bad idea. I would like to keep as other implementations in services.